File: C/CalculadoraSimples.c
#include <stdio.h>

int main () 
{
    int A;
    int B;
    int soma,sub,mult;
    float div,num1,num2;
    scanf("%d",&A);
    scanf("%d",&B);
    soma = A + B;
    sub = A - B;
    mult = A * B; 
    if (A != 0 && B != 0)
    {
        num1 = A;
        num2 = B;
        div = num1 / num2;
        printf("SOMA = %d\n",soma);
        printf("SUBTRACAO = %d\n",sub);
        printf("MULTIPLICACAO = %d\n",mult);
        printf("DIVISAO = %.2f\n", div);
    }
    else if (B == 0)
    {
       printf("SOMA = %d\n",soma);
       printf("SUBTRACAO = %d\n",sub);
       printf("MULTIPLICACAO = %d\n",mult);
       printf("DIVISAO = INDETERMINADO\n"); 
    }  
    return 0;
}

File: Java/CalculadoraSimples.java
import java.util.Scanner;

public class Main 
{
    public static void main (String [] args)
	{
		int n1, n2, soma, mult, sub;
		float div;
		Scanner in = new Scanner ( System.in );
		n1 = in.nextInt();
		n2 = in.nextInt();
		soma = n1 + n2;
		mult = n1 * n2;
		sub = n1 - n2;
		div = n2 / n1;
		
		if ( n1 == 0 || n2 == 0)
		{
			System.out.printf("SOMA = %d\n",soma);
		    System.out.printf("SUBTRACAO = %d\n",sub);
		    System.out.printf("MULTIPLICACAO = %d\n",mult);
			System.out.printf("DIVISAO = INDETERMINADO\n");
		}
		
		else
		{
			System.out.printf("SOMA = %d\n",soma);
		    System.out.printf("SUBTRACAO = %d\n",sub);
		    System.out.printf("MULTIPLICACAO = %d\n",mult);
		    System.out.printf("DIVISAO = %.2f\n",div);
		}
		
		in.close();	 
	}
}
